CONTRIBUTING.md

```diff
@@ -4,29 +4,51 @@
 [pr]: /compare
 [style]: https://standardjs.com/
 [code-of-conduct]: CODE_OF_CONDUCT.md
+[slack]: https://probot-slackin.herokuapp.com/
```

Comment:
Do you think this is worthwhile to include here since not all app creators might want to redirect to our "official" slack?